atlassian_jwt_auth/contrib/django/decorators.py

```diff
@@ -73,11 +73,15 @@ def requires_asap_wrapper(request, *args, **kwargs):
     return requires_asap_decorator
 
 
-def _get_verifier():
+def _get_verifier(subject_should_match_issuer=None):
     """Return a verifier for ASAP JWT tokens based on settings"""
     retriever_cls = getattr(settings, 'ASAP_KEY_RETRIEVER_CLASS',
                             atlassian_jwt_auth.HTTPSPublicKeyRetriever)
     retriever = retriever_cls(
         base_url=getattr(settings, 'ASAP_PUBLICKEY_REPOSITORY')
     )
-    return atlassian_jwt_auth.JWTAuthVerifier(retriever)
+    if subject_should_match_issuer is None:
+        subject_should_match_issuer = getattr(
```

Comment:
IMHO if `subject_should_match_issuer` is `None` and `settings` has no `ASAP_SUBJECT_SHOULD_MATCH_ISSUER` attribute you should should not provide a value for `subject_should_match_issuer` so that the default value for in the `JWTAuthVerifier` for `subject_should_match_issuer` will be used.

Reply:
Yes, absolutely.